src/commands/registries/azure/deployImageToAca.ts

```diff
@@ -4,10 +4,10 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { IActionContext, nonNullProp, UserCancelledError } from '@microsoft/vscode-azext-utils';
+import { parseDockerLikeImageName } from '@microsoft/vscode-container-client/lib/clients/DockerClientBase/parseDockerLikeImageName';

```

Comment:
I don't feel great about importing from here (i.e. from deep in the package instead of `index.js`), but I also don't feel great about exporting that parse utility. I'd like to hear others' thoughts on it.

Reply:
i'd vote not exporting it

Reply:
Maybe we should consider exposing it as a static method on DockerClientBase so `import { DockerClientBase } from ...` and `DockerClientBase.parseDockerLikeImageName(...)`

Reply:
I think I've changed my mind...I think exporting it is the way to go. We're already exporting some stuff from utils anyway.

Reply:
https://github.com/microsoft/vscode-docker-extensibility/pull/169